Dockerfile

```diff
@@ -59,56 +79,104 @@ RUN set -eux; \
     \
     # tools used by Pimcore
     apt-get install -y \
-        ffmpeg ghostscript jpegoptim exiftool poppler-utils optipng pngquant webp graphviz locales locales-all git; \
+        exiftool \
+        ffmpeg \
+        ghostscript \
+        git \
+        graphviz \
+        jpegoptim \
+        locales \
+        locales-all \
+        optipng \
+        pngquant \
+        poppler-utils \
+        webp \
+    ; \
     \
-    # dependencies fór building PHP extensions
-    apt-get install -y libwebp-dev libfreetype6-dev; \
+    # dependencies for building PHP extensions
+    apt-get install -y \
+        libfreetype6-dev \
+        libwebp-dev \
+    ; \
     \
     # ImageMagick
-    apt-get install -y imagemagick-7 libmagickwand-7-dev; \
+    apt-get install -y \
+        imagemagick-7 \
+        libmagickwand-7-dev \
+    ; \
     \
-    docker-php-ext-configure gd -enable-gd --with-freetype --with-jpeg --with-webp; \
+    docker-php-ext-configure gd --enable-gd --with-freetype --with-jpeg --with-webp; \
     docker-php-ext-install gd; \
     \
-    pecl install -f apcu redis imagick; \
-    docker-php-ext-enable redis apcu imagick; \
+    pecl install -f \
+        apcu \
+        imagick \
+        redis \
+    ; \
+    docker-php-ext-enable \
+        apcu \
+        imagick \
+        redis \
+    ; \
+    \
+    build-cleanup.sh; \
+    \
     ldconfig /usr/local/lib; \
```

Comment:
`ldconfig` runs now after `build-cleanup.sh`. Are you aware of this?

Reply:
Sure, and it actually doesn't do anything. But I didn't want to remove/rewrite everything. This might be a left over after a previous refactoring.

Reply:
Alright! Thanks